src/replace_scp.py

```diff
@@ -0,0 +1,89 @@
+"""Replace SCP.
+
+Purpose:
+    Replace DETACH_SCP_ID SCP with ATTACH_SCP_ID SCP defined in the associated env vars
+Environment Variables:
+    LOG_LEVEL: (optional): sets the level for function logging
+            valid input: critical, error, warning, info (default), debug
+    DETACH_SCP_ID: id of the policy to detach
+    ATTACH_SCP_ID: id of the new policy to attach
+    ASSUME_ROLE_NAME: Name of role to assume
+"""
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+import logging
+import os
+import sys
+
+from aws_lambda_powertools import Logger
+import boto3
+
+# Standard logging config
+LOG_LEVEL = os.environ.get("LOG_LEVEL", "info")
+LOG = Logger(
+    service="new_account_replace_scp",
+    level=LOG_LEVEL,
+    stream=sys.stderr,
+    location="%(name)s.%(funcName)s:%(lineno)d",
+    timestamp="%(asctime)s.%(msecs)03dZ",
+    datefmt="%Y-%m-%dT%H:%M:%S",
+)
+
+ATTACH_SCP_ID = os.environ["ATTACH_SCP_ID"]
+DETACH_SCP_ID = os.environ["DETACH_SCP_ID"]
+
+# Get client
+org_client = boto3.client("organizations")
+
+@LOG.inject_lambda_context(log_event=True)
+def lambda_handler(event, context):  # pylint: disable=unused-argument
+    """Replace scp policy lambda handler."""
+    account_id = event["detail"]["recipientAccountId"]
+    replace_scp(account_id)
+
+
+def replace_scp(account_id):
```

Comment:
When the event is `CreateOrganizationalUnit`, the target will be an OU ID, not an Account ID...